hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.metrics;
+
+/**
+ * Class holding constants for metric names, prefixes, discriminators etc.
+ * <p/>
+ *
+ * IMPORTANT NOTE:
+ * The constants in this file are meant to be stable and can be changed
+ * between minor releases only if at all. Changing any of the constants
+ * in this class might break Management Center or any metrics consumer
+ * code.
+ */
+public final class MetricDescriptorConstants {
+
+    // ===[CACHE]=======================================================
+    public static final String CACHE_PREFIX = "cache";
+    public static final String CACHE_DISCRIMINATOR_NAME = "name";
```

Comment:
Just wondering - instead of having separate `CACHE_DISCRIMINATOR_NAME`, `EXECUTOR_DISCRIMINATOR_NAME`, `MAP_DISCRIMINATOR_NAME`, `GENERAL_DISCRIMINATOR_NAME`, wouldn't it be better to have just one - `DISCRIMINATOR_NAME`? The same maybe for other discriminators.
But this is minor and probably just a mater of taste.

Reply:
No strong reason for having separate discriminator names. I wanted to keep all related constants around the same section, maybe it's easier to look for what we have for a particular DS, service, etc. There is a `GENERAL_DISCRIMINATOR_NAME` though, which is the only exception from the above "guideline". That's because it is used in a general place. If no strong reasons for changing it, I'd prefer to keep it as is.